pyracf/common/security_admin.py

```diff
@@ -157,32 +179,59 @@ def _make_request(
                 security_request.dump_request_xml(encoding="utf-8"),
                 secret_traits=self.__secret_traits,
             )
+        request_xml = self.__logger.redact_request_xml(
+            security_request.dump_request_xml(encoding="utf-8"),
+            secret_traits=self.__secret_traits,
+        )
         if self._generate_requests_only:
-            request_xml = self.__logger.redact_request_xml(
-                security_request.dump_request_xml(encoding="utf-8"),
-                secret_traits=self.__secret_traits,
-            )
             self.__clear_state(security_request)
             return request_xml
         result_xml = self.__logger.redact_result_xml(
```

Comment:
Given that `result_xml` may actually not be XML in some situations, it may make more sense to call this variable `result` to avoid confusion about `DownstreamFatalError`.

Reply:
agree, this is appropriate. change made.